api/envoy/api/v2/route/route.proto

```diff
@@ -109,6 +109,10 @@ message VirtualHost {
   // specific; see the :ref:`HTTP filter documentation <config_http_filters>`
   // for if and how it is utilized.
   map<string, google.protobuf.Struct> per_filter_config = 12;
+
+  // Decides whether the x-envoy-attempt-count header should be included in upstream
```

Comment:
Can you add to https://www.envoyproxy.io/docs/envoy/latest/configuration/http_filters/router_filter#http-headers-set and cross link?